<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="mainLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QStackedWidget" name="mapStackedWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="map">
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="mapLayout"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="plot">
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="plotLayout"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="simulationLayout">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QPushButton" name="pauseSim">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normalon>:/ui_icons/control-pause.png</normalon>
           </iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="startSim">
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="icon">
           <iconset>
            <normalon>:/ui_icons/control-play.png</normalon>
           </iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="sliderTick">
          <property name="maximum">
           <number>0</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinTick">
          <property name="maximum">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelUpperBoundTick">
          <property name="text">
           <string>/ 0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelTickUnit">
          <property name="text">
           <string>frames</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>712</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionLoadFromDB"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuPreferences">
    <property name="title">
     <string>Preferences</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
   <addaction name="menuPreferences"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionLoadFromDB"/>
   <addaction name="separator"/>
   <addaction name="actionToggleHideLeftWidget"/>
   <addaction name="actionToggleStaticElements"/>
   <addaction name="actionToggleDynamicPage"/>
   <addaction name="separator"/>
   <addaction name="actionTogglePointTracker"/>
   <addaction name="actionToggleLog"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="leftWidget">
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string/>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QStackedWidget" name="leftStackedWidget">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="geospatialWidget">
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="1" column="0">
          <widget class="QSplitter" name="geoSplitter">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="handleWidth">
            <number>7</number>
           </property>
           <property name="childrenCollapsible">
            <bool>true</bool>
           </property>
           <widget class="QTreeWidget" name="geospatialTree">
            <column>
             <property name="text">
              <string>Type</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Number</string>
             </property>
            </column>
           </widget>
           <widget class="QTableWidget" name="propertiesTable">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(242, 242, 242);
border: none;</string>
            </property>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Property Name</string>
             </property>
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Property Value</string>
             </property>
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
            </column>
           </widget>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLineEdit" name="geospatialSearchField">
           <property name="styleSheet">
            <string notr="true">background: url(:/ui_icons/search-icon.png);
background-position: right;
background-repeat: no-repeat;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="dynamicWidget">
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <widget class="QLabel" name="dynamicLabel">
           <property name="font">
            <font>
             <italic>true</italic>
            </font>
           </property>
           <property name="text">
            <string>Select an agent on screen.</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QTableWidget" name="dynamicTable">
           <property name="styleSheet">
            <string notr="true">border: none;</string>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <column>
            <property name="text">
             <string>Property</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Value</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="logWidget">
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item row="2" column="1">
          <widget class="QTextEdit" name="logTextEdit">
           <property name="acceptRichText">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="logtopLayout">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item alignment="Qt::AlignRight">
            <widget class="QPushButton" name="saveLogButton">
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="styleSheet">
              <string notr="true">padding: 3px;
border: none;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normalon>:/ui_icons/disk-black.png</normalon>
              </iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>:/ui_icons/blue-folder-open-document-text.png</normaloff>
     <normalon>:/ui_icons/open.png</normalon>:/ui_icons/blue-folder-open-document-text.png</iconset>
   </property>
   <property name="text">
    <string>open</string>
   </property>
   <property name="toolTip">
    <string>Open an existing Simmobility file</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/ui_icons/about.png</normaloff>:/ui_icons/about.png</iconset>
   </property>
   <property name="text">
    <string>about</string>
   </property>
   <property name="toolTip">
    <string>About iSimGUI</string>
   </property>
  </action>
  <action name="actionToggleHideLeftWidget">
   <property name="icon">
    <iconset>
     <normalon>:/ui_icons/application-resize-full.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>toggleHideLeftWidgets</string>
   </property>
   <property name="toolTip">
    <string>Hide left widgets</string>
   </property>
  </action>
  <action name="actionToggleStaticElements">
   <property name="icon">
    <iconset>
     <normalon>:/ui_icons/search_map.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>toggleStaticElements</string>
   </property>
   <property name="toolTip">
    <string>search static elements</string>
   </property>
  </action>
  <action name="actionToggleDynamicPage">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/ui_icons/Car_Icon.png</normaloff>
     <normalon>:/ui_icons/bus_route.png</normalon>:/ui_icons/Car_Icon.png</iconset>
   </property>
   <property name="text">
    <string>toggleDynamicPage</string>
   </property>
   <property name="toolTip">
    <string>view dynamic components</string>
   </property>
  </action>
  <action name="actionLoadFromDB">
   <property name="icon">
    <iconset>
     <normalon>:/ui_icons/database_search.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>loadFromDB</string>
   </property>
   <property name="toolTip">
    <string>Load data from database</string>
   </property>
  </action>
  <action name="actionToggleLog">
   <property name="icon">
    <iconset>
     <normalon>:/ui_icons/log.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>toggleLog</string>
   </property>
   <property name="toolTip">
    <string>Show log</string>
   </property>
  </action>
  <action name="actionTogglePointTracker">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/ui_icons/pointTracker.png</normaloff>:/ui_icons/pointTracker.png</iconset>
   </property>
   <property name="text">
    <string>togglePointTracker</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
